plugins/jobs/web_client/views/JobGraphWidget.js

```diff
@@ -110,13 +108,11 @@ const JobGraphWidget = View.extend({
             config.height = this.$('.g-jobs-graph').height();
             config.data[0].values = vegaData;
             config.scales[1].type = this.yScale;
-            config.scales[2].domain = jobs.map(job => job.get('_id'));
-            config.scales[2].range = jobs.map(job => {
-                let datetime = moment(job.get('updated')).format('MM/DD');
-                return datetime;
-            });
-            config.scales[3].domain = jobs.map(job => job.get('_id'));
-            config.scales[3].range = jobs.map(job => job.get('title'));
+            config.scales[2].domain = this.collection.pluck('_id');
+            config.scales[2].range = this.collection.map(
+                (job) => moment(job.get('updated')).format('MM/DD'));
+            config.scales[3].domain = this.collection.pluck('_id');
```

Comment:
We use Backbone Collections for a reason, casting to an Array removes all of the utilities and benefits:
* Collections have several internal structures to improve the performance of methods like `.at`
* Casting to an Array takes some constant amount of initial overhead and additional memory
* It makes a copy of the object, making the code harder to reason about and maintain (e.g. if we ever wanted to start mutating the Collection)

IMHO, understanding Backbone is a hard requirement for programming Girder's front end. All notions about developer readability should be predicated on them having adequate Backbone knowledge.